File: src/org/cappuccino_project/ide/intellij/plugin/psi/interfaces/ObjJConditionalStatement.java
package org.cappuccino_project.ide.intellij.plugin.psi.interfaces;

import org.cappuccino_project.ide.intellij.plugin.psi.ObjJExpr;

public interface ObjJConditionalStatement extends ObjJCompositeElement {
    ObjJExpr getConditionalExpression();
}

File: src/org/cappuccino_project/ide/intellij/plugin/psi/interfaces/ObjJConditionalStatement.kt
package org.cappuccino_project.ide.intellij.plugin.psi.interfaces

import org.cappuccino_project.ide.intellij.plugin.psi.ObjJExpr

interface ObjJConditionalStatement : ObjJCompositeElement {
    val conditionalExpression: ObjJExpr
}
